examples/thread-duell/main.c

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (C) 2020 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <thread.h>
+#include <sched.h>
+
+#include <math.h>
+
+#include "xtimer.h"
+
+#include "sched_round_robin.h"
+
+#define PRINT_STEPS 10
+#define WORK_SCALE  1000
+
+void bad_wait(uint64_t us)
```

Comment:
this and `nice_wait()` should be static, and prefixed with `_`.

Reply:
```
RIOT/examples$ rg static |wc -l
400
RIOT/examples$ rg 'static.* _'|wc -l
136
RIOT/examples$ rg 'static.* \*_'|wc -l
15
```
151 of 400 declared statics in examples have `_ or *_`  in front